extensions/sonarqube/overview.md

```diff
@@ -1,50 +1,59 @@
 
-This extension provides build tasks that you can add in your build definition. All branches and pull-requests are automatically analyzed, allowing you to discover early any bug or vulnerability in the code.
+SonarQube Commercial Editions tightly integrate with your Azure DevOps environment and analyze branches and Pull Requests so your team spots and resolves issues before you merge to master. Analysis results are published right in your build summary! 
 
-*Note: this extension is officially supported for use with Azure DevOps Server (formerly TFS). Ongoing interoperability with Azure DevOps Services (cloud-based) is not assured.*
+This extension provides tasks that you incorporate into your build definition(s) to enable additional SonarQube functionality in Azure DevOps environments.
 
-## About the SonarQube Azure DevOps Marketplace Extension
-This extension provides the following features:
-* A dedicated **SonarQube EndPoint** to define the SonarQube server to be used.
-* Three build tasks to get your projects analyzed easily:
-  * **Prepare Analysis Configuration** task, to configure all the required settings before executing the build. This task is mandatory. In case of .NET solutions or Java projects, this tasks helps to integrate seamlessly with MSBuild, Maven and Gradle tasks.
-  * **Run Code Analysis** task, to actually execute the analysis of the source code. Not required for Maven or Gradle projects.
-  * **Publish Quality Gate Result** task, to display the quality gate status in the build summary. This tasks is optional, as it can increase the overall build time.
-  * Analysis of the branches and the **pull requests** of your projects
+_Note: this extension is officially supported for use with Azure DevOps Server (formerly TFS). Ongoing interoperability with Azure DevOps Services (cloud-based) is not assured._
 
-**Note for TFS installations older than TFS 2017 Update 2**: to install the extension, please follow instructions
-available on the ["SonarQube Extension 3.0" documentation page](https://docs.sonarqube.org/display/SCAN/SonarQube+Extension+3.0).
+## Benefits of the SonarQube Azure DevOps Marketplace Extension
 
-## Highlighted Features
-### Seamless Integration with .Net solutions
-The analysis of C# and VB. Net solution is really straightforward since it only requires adding the two **Prepare Analysis Configuration** and **Run Code Analysis** tasks to your build definition.
+### Quality Gate™ Status Publishing
+A Quality Gate™ is a Pass/Fail status indicator that clearly lets you know if your code is clean and safe. SonarQube comes with a default Quality Gate called SonarWay™ that's built-in and ready to use. When you see a 'Green' Quality Gate, you know that your application is releasable and your team is hitting the mark! Using this extension, SonarQube publishes Quality Gate status right to your build summary.
```

Comment:
It's actually called "Sonar way"
Also, no need to repeat the ^TM^ in this paragraph, right?